content/doc/book/managing/plugins.adoc

```diff
@@ -225,6 +225,67 @@ The configuration(s) created by the disabled plugin behave as if the plugin
 were uninstalled, insofar that they result in warnings on boot but are
 otherwise ignored.
 
+==== Using the Jenkins CLI
+
+It is also possible to enable and disable plugins using the <<cli#,Jenkins CLI>>.
+Since https://jenkins.io/changelog/#v2.136[Jenkins 2.136], administrators may 
+invoke the `enable-plugin` command to re-enable a collection of plugins which 
+were previously disabled and since https://jenkins.io/changelog/#v2.151[Jenkins 2.151] 
+the `disable-plugin`, which provides a command to disable installed plugins, is included.
+
+The `enable-plugin` command receives a list of plugins to be enabled. If any of 
+the plugins have one or more dependencies, these plugins and their dependencies
+will be enabled.
+
+[source]
+----
+java -jar jenkins-cli.jar -s http://localhost:8080/ enable-plugin PLUGIN ... [-restart]
+
+Enables one or more installed plugins transitively.
+
+ PLUGIN   : Enables the plugins with the given short names and their
+            dependencies.
+ -restart : Restart Jenkins after enabling plugins.
+----
+
+The `disable-plugin` command receives a list of plugins to be disabled. The 
+output will display messages for both successful and failed operations. In case
+that only the error messages are desired, the `-quiet` option can be specified.
+How to deal with other plugins which depend on one or more of the plugins 
+passed as parameter may be specified with the `-strategy` option.
+
+[source]
+----
+java -jar jenkins-cli.jar -s http://localhost:8080/ disable-plugin PLUGIN ... [-quiet (-q)]
+[-restart (-r)] [-strategy (-s) strategy]
+
+Disable one or more installed plugins.
+Disable the plugins with the given short names. You can define how to proceed with the 
+dependant plugins and if a restart after should be done. You can also set the quiet mode 
+to avoid extra info in the console.
+
+ PLUGIN                  : Plugins to be disabled.
+ -quiet (-q)             : Be quiet, print only the error messages
+ -restart (-r)           : Restart Jenkins after disabling plugins.
+ -strategy (-s) strategy : How to process the dependant plugins. 
+                           - none: if a mandatory dependant plugin exists and
+                           it is enabled, the plugin cannot be disabled
+                           (default value).
+                           - mandatory: all mandatory dependant plugins are
+                           also disabled, optional dependant plugins remain
+                           enabled.
+                           - all: all dependant plugins are also disabled, no
+                           matter if its dependency is optional or mandatory.
+----
+
+[CAUTION]
+====
+In the same way than enabling and disabling plugins from the UI need a restart 
+to complete the process, the changes made with the CLI commands will take effect
+once Jenkins is restarted. The `-restart` option forces a safe restart of the 
+instance once the command has successfully finished, so the changes will be 
```

Comment:
I would drop the comma in "finished, so". It isn't needed and disrupts the flow.